onnxruntime/core/graph/graph.cc

```diff
@@ -1793,31 +1790,57 @@ Status Graph::VerifyNodeAndOpMatch() {
                                                                                            logger_));
       node.SetFunctionBody(*function_container_.back());
     }
-
+    NodeProto node_proto;
+    node.ToProto(node_proto);
     if (!node.Op()) {
       try {
-        checker::check_node(node_proto, ctx, lsc);
-      } catch (const std::exception& ex) {
-        return ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_GRAPH, "This is an invalid model. Error in Node:", node_name, " : ", ex.what());
-      }
+        auto iter2 = domain_to_version_.find(domain);
+        if (iter2 == domain_to_version_.end()) {
```

Comment:
Could this logic maybe be in utils in a standalone function called CheckNode? Would be a bit clearer that the multiple steps here are all part of that check, and there could be a comment saying that it replaces using onnx::check_node and why, so that info is captured.

Reply:
Will do it.